src/glue/resources/schema_mapping.json

```diff
@@ -5,5 +5,9 @@
     "https://sage-bionetworks.github.io/mobile-client-json/schemas/v1/TappingResultObject.json": "TappingResultObject_v1",
     "https://sage-bionetworks.github.io/mobile-client-json/schemas/v2/WeatherResult.json": "WeatherResult_v1",
     "https://sage-bionetworks.github.io/mobile-client-json/schemas/v2/AudioLevelRecord.json": "AudioLevelRecord_v1",
-    "sharedSchema_for_Gradual_Onset": "sharedSchema_for_Gradual_Onset_v1"
+    "sharedSchema_for_Gradual_Onset": "sharedSchema_for_Gradual_Onset_v1",
+    "schemas/v0/3DRotation.json": "3DRotation_v1",
```

Comment:
Is there a reason why this new schema mapping url doesn't have the full URL path like `https://sage-bionetworks.github.io/mobile-client-json/schemas/...`

Reply:
No reason. The `$id` can be any value as long as it's unique.